dnaStreaming/config.py

```diff
@@ -49,7 +49,8 @@ def _validate(self):
             raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), self.customer_config_path)
 
         if not os.access(self.customer_config_path, os.R_OK):
-            raise Exception('Encountered permission problem reading file from path \'{}\'.'.format(self.customer_config_path))
+            raise Exception('Encountered permission problem reading file from path \'{}\'.'.
```

Comment:
The max line length defined in .flake8 is 180, and this line is 127 characters length, I don't think this change is necessary

Reply:
oh yeah, I think a previous team member set a limit to 120 but we haven't enforced it in a while. I can undo these changes